source/Präzedenz.R

```diff
@@ -1,4 +1,4 @@
-# Beispiele für Präzendenz:
+# Beispiele für Präzedenz:
```

Comment:
Wird diese Datei überhaupt innerhalb des Skripts genutzt? Egal, ist mir nur spontan ins Auge gefallen^^

Reply:
Ne das soll als ein neuer Abschnitt in Kapitel 2 eingefügt werden